conf/routes

```diff
@@ -64,7 +64,7 @@ GET         /triage/:id                                        @femr.ui.controll
 GET         /triage                                            @femr.ui.controllers.TriageController.indexGet()
 POST        /triage                                            @femr.ui.controllers.TriageController.indexPost(id: Integer)
 POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer)
-
+POST        /history/patient/:id                              @femr.ui.controllers.TriageController.deleteEncounterPost(id: Integer,ecounterid: Integer)
```

Comment:
This doesn't really fit in well with the Triage section, perhaps better in the History section next to the other encounter routes. Since this is an action performed on an encounter and not a patient you will want to use the same template "/history/encounter/action/:id" like the others:

https://github.com/FEMR/femr/blob/master/conf/routes#L48